app/scripts/views/sign_in.js

```diff
@@ -310,4 +325,11 @@ function (Cocktail, p, BaseView, FormView, SignInTemplate, Session,
   );
 
   return View;
+
+  function interTabSignInHandler (event) {
```

Comment:
Do you think it makes sense to take this functionality, add it to a mixin, and hook it up to `/signup` and `/reset_password` too? I'm unsure about how the `/confirm_*` screens should act.